packages/react-aria-components/stories/DatePicker.stories.tsx

```diff
@@ -16,10 +16,13 @@ import React from 'react';
 import styles from '../example/index.css';
 
 export default {
-  title: 'React Aria Components'
+  title: 'React Aria Components',
+  argTypes: {
+    isFixedWeeks: {control: 'boolean'}
+  }
```

Comment:
I noticed in the DatePicker stories that the dates outside the current month seem to be non-interactive (aka clicking on them doesn't select the date). This seems to be because the dates that fall outside the month are judged as disabled via https://github.com/adobe/react-spectrum/blob/fd3f09c798da23bf2148ba11f65e8856f8af9de6/packages/%40react-stately/calendar/src/useCalendarState.ts#L307. For this case, we should probably make those days considered as valid dates if `isFixedWeeks` is true but will bring this up to the team

Reply:
It seems that in previous discussions, it was concluded that enabling dates outside the month/visible range would introduce too many UX challenges.

> The team discussed this again at length today, and we decided that there are too many UX problems with enabling dates outside the month / visible range.

https://github.com/adobe/react-spectrum/issues/3257#issuecomment-1644757484

Reply:
thanks for the reminder, I forgot about that investigation we did last year. In that case I'm not super thrilled about adding this behavior as a whole then, doesn't really feel very useful if those dates would always be disabled + dimmed. I guess it wouldn't hurt since it could stabilize the number of cells shown but I feel like users may then want to have those dates be interactive...

Reply:
You're right, making the displayed dates interactive could indeed be more useful. Perhaps focusing on a single datepicker, rather than a range picker for selecting multiple dates, would be a better approach this time.